onboarding/matplot.py

```diff
@@ -5,3 +5,15 @@
 - Be careful of data points that don't exist
 - Will require reading from CSV, can experiment with pandas
 '''
+import matplotlib.pyplot as plt
+import csv
+
+with open('test_data.csv') as csv_file:
+    content = csv.reader(csv_file)
+    for row in content:
+        print(row)
+        x = row[0]
+        for i in range(1, 4):
+            plt.scatter(x, row[i])
```

Comment:
Can you make 4 separate line graphs for this one. The scatters look really bad.

Reply:
should also be fixed now